### ISSUE TITLE
Fallback matching

### ISSUE THREAD

**Issue by Kommynct** [philj56/tofi#96]
I see your recent prefix matching patch, and I was thinking, I would like to use prefix, but i'd like it if tofi did fuzzy if it can't find any results using the other method.

**Comment by philj56** [comment]
Yeah that's not a bad idea, I'll give it a go. It'd be nice to be able to specify a list, so you could do e.g. `--matching-algorithm=prefix,normal,fuzzy`.

On that note, that's also probably a good use for real Levenshtein distance matching. I implemented it on a separate branch, but decided it was kinda useless as it doesn't work unless you've typed nearly the entire word. Having it as a fallback could be cool though.
